draft-ietf-quic-recovery.md

```diff
@@ -199,11 +199,18 @@ RTT is calculated when an ACK frame arrives by computing the difference
 between the current time and the time the largest newly acked packet was sent.
 If no packets are newly acknowledged, RTT cannot be calculated. When RTT is
 calculated, the ack delay field from the ACK frame SHOULD be subtracted from
-the RTT as long as the result is positive.
+the RTT as long as the result is larger than the Min RTT.  If the result is
+smaller than the min_rtt, the RTT should be used, but the ack delay field
+should be ignored.
 
 Like TCP, QUIC calculates both smoothed RTT and RTT variance as specified in
 {{?RFC6298}}.
 
+Min RTT is the minimum RTT measured over the connection, prior to adjusting
```

Comment:
A connection might be long lived, so it kind of feels weird to have this value calculated over the whole lifetime of the connection. Maybe this is ok, since we're only using Min RTT as a sanity check for the ACK delay sent by the peer.

Reply:
BBR does use windowed min, but I think this is a fairly standard definition of min_rtt.

Reply:
I think it's ok... we can get fancy here (with windowed min), but I think this is good for a simple safeguard.